dts.config.js

```diff
@@ -0,0 +1,30 @@
+const libraries = {
+    inlinedLibraries: [
+        '@pixi/colord',
+        'earcut',
+        'eventemitter3',
+    ],
+};
+
+const output = {
+    inlineDeclareGlobals: true,
+    exportReferencedTypes: false,
+    umdModuleName: 'PIXI',
```

Comment:
https://github.com/pixijs/pixijs/pull/10178#discussion_r1477131392 this might be still applicable, but I think it should be validated in manual tests what's more appropriate (a bit more context on `export as namespace` or what `umdModuleName` does https://stackoverflow.com/questions/44847749/explain-export-and-export-as-namespace-syntax-in-typescript).

Reply:
I've been testing this out and the `export as namespace` behaviour seems to be working for every use case i've tried so far. So happy to leave it as is and we can change later if needed